addons/web/static/src/views/form/form_controller.scss

```diff
@@ -255,6 +257,7 @@
     .o_form_sheet {
         --formView-sheet-padding-y: #{map-get($spacers, 3)};
         --formView-sheet-padding-x: #{map-get($spacers, 3)};
+        --alert-margin-bottom: initial;
```

Comment:
Thanks, I didn't know about the behaviour of `initial` when used for custom properties 👍